sgkit/tests/test_popgen.py

```diff
@@ -309,9 +345,10 @@ def test_Fst__windowed(sample_size, n_cohorts, chunks):
         )  # scikit-allel has final window missing
 
 
+@pytest.mark.skip("Bug in Tajimas D for n=100?")
```

Comment:
Can we run the cases that work in the meantime?

Reply:
Yes - I was being lazy. How do you skip a particular example with parametrize?

Reply:
I don't know if it's possible. But we could remove `100` from the list, as long as we have an issue open (which we do: #523).

Reply:
Ah, actually no, it's failing for all n > 2, so it's probably simplest to keep skipping it rather than giving a false sense of security here.

Reply:
OK, I agree it's best to skip then.